packages/cactus-plugin-htlc-eth-besu-erc20/src/main/typescript/generated/openapi/typescript-axios/git_push.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/sh
```

Comment:
@jordigiam Please ignore this file in the openapi generator ignore file + make sure this file is deleted from the change

Reply:
Done